app/src/main/java/de/upb/hip/mobile/activities/ExhibitDetailsActivity.java

```diff
@@ -617,6 +646,20 @@ private void updatePlayPauseButtonIcon() {
 
     }
 
+    /** Hides the audio action in the toolbar */
+    private void hideAudioAction() {
+        View audioIcon = findViewById(R.id.action_audio);
+        if (audioIcon != null)
+            audioIcon.setVisibility(View.GONE);
+    }
+
+    /** Shows the audio action in the toolbar */
+    private void showAudioAction() {
+        View audioIcon = findViewById(R.id.action_audio);
+        if (audioIcon != null)
+            audioIcon.setVisibility(View.VISIBLE);
+    }
+
```

Comment:
hideAudioAction and showAudioAction are not needed anymore (they have been replaced by displayAudioAction).